Dockerfile

```diff
@@ -1 +1,51 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+ENV PENTAHO_CE_VERSION=8.1.0.0-365
+ENV Mysql_Connector_Version=8.0.24
+ENV Mariadb_Connector_Version=2.2.6
+ENV Arkcase_Pre_Auth_Version=4-1.1.1
+ARG RESOURCE_PATH="artifacts"
 
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_pentaho \
+      MAINTAINER="Armedia LLC"
+
+#-----------PENTAHO SETUP-------------------------------------
+RUN mkdir -p /home/pentaho/data/pentaho &&\
+    mkdir -p /home/pentaho/app/pentaho && \
+    mkdir -p /home/pentaho/tmp/pentaho && \
```

Comment:
I think this directory structure is inspired from the Pentaho Ansible scripts, but it makes little sense in a Docker image. Please follow what Naveen did and put all Pentaho-related files under `/opt/pentaho`.

Reply:
Our Internal pentaho-ce software might use directory stricture like above. So that is the reason i haven't changed the directory path

Reply:
@dheeraj-armedia Sorry for the very late response. I'm sorry, but I don't understand your reply. Could you please elaborate? Thanks